include/oboe/Definitions.h

```diff
@@ -495,6 +495,25 @@ namespace oboe {
         int64_t timestamp; // in nanoseconds
     };
 
+    class OboeGlobals {
```

Comment:
I'm fine with these being set here. I wonder if the default stream values should be moved into this class as well.

Reply:
Add getters/setters for the default stream values e.g. `setOpenSLESDefaultSampleRate` etc 
Add comment saying that direct access to `DefaultStreamValues` is deprecated.

Reply:
One other thing. It might be better to name this `Globals` rather than `OboeGlobals` since this is already in the `oboe` namespace.